sources/packages/web/src/components/layouts/Institution/sidebar/HomeSideBar.vue

```diff
@@ -26,6 +26,7 @@
             :title="location.label"
             :prepend-icon="location.icon"
             :value="location.value"
+            :data-cy="`location-${index}`"
```

Comment:
Does it really help the tests?
If an automation script would like to select the locations it will probably be selecting all the are `data-cy="location"`.

Reply:
it was a leftover. removing. I was showing @velang01 how to add the data-cy.